### ISSUE TITLE
Google colab integration

### ISSUE THREAD

**Issue by vainaijr** [damianavila/RISE#478]
Hello,

Is it possible to make slideshow from google colab notebook, because on google colab we have access to cloud GPU, TPU, it would be convenient to make slides directly from google colab.

Thanks

**Comment by damianavila** [comment]
It should be possible, but I have not played with Colab recently, so I am not sure how is the status of running nbextensions on Colab.

**Comment by hugocool** [comment]
hi! is there any progress on this? 
Our company is a big colab user, and we use RISE all the time internally. But I love how easy it is to share colab notebooks. If it could also be that easy to share RISE presentations based on the notebook, that would be amazing!

**Comment by g-eorge** [comment]
:+1: Colab integration would be very useful.

**Comment by Naereen** [comment]
I vote :+1: * 100 for RISE integration on Colab!

Just wanted to share https://notebook.basthon.fr/ which is a Jupyter + Python working entirely in javascript!
It's fully open source (site & doc in French but code & devs on https://framagit.org can speak English).
The Python code is interpreted using [Pyodide](https://github.com/iodide-project/pyodide/), it works amazingly well!

And... it has RISE installed! :clap:

**Comment by damianavila** [comment]
Ja... I actually played with pyiodide as well a long time ago to get a notebook running python on JS.
And I think there are others out there with the same idea, @Naereen. 
* https://github.com/deathbeds/jyve
* https://github.com/jtpio/p5-notebook/pull/52
Some links in case you do not know them...

> And... it has RISE installed! 👏

I do not see RISE there... but I would love to see it :wink:

**Comment by Naereen** [comment]
Hi @damianavila, 
Thanks for the links!

On <https://notebook.basthon.fr/> you might have to wait a few seconds to see the RISE button appear, here is a short video demo (GIF):
![Peek 08-03-2021 22-38](https://user-images.githubusercontent.com/11994719/110385328-1eaf8580-805f-11eb-97aa-72683e5c2da8.gif)

**Comment by vivianamarquez** [comment]
> I vote 👍 * 100 for RISE integration on Colab!
> 
> Just wanted to share https://notebook.basthon.fr/ which is a Jupyter + Python working entirely in javascript! It's fully open source (site & doc in French but code & devs on https://framagit.org can speak English). The Python code is interpreted using [Pyodide](https://github.com/iodide-project/pyodide/), it works amazingly well!
> 
> And... it has RISE installed! 👏

This is incredibly useful! I'm a teacher and I would use it all the time if it was in English.
